.coveragerc

```diff
@@ -0,0 +1,17 @@
+[run]
+source = ZConfig
+
+[report]
+# Coverage is run on Py2 and Py3
+# on POSIX platforms, so omit Windows-specific
+# things. Omit the main methods of test files.
+exclude_lines =
+    pragma: no cover
+    pragma no cover
```

Comment:
I guess there are no occurences of 'pragma no cover' anymore so that can go away.